code/modules/research/techweb/all_nodes.dm

```diff
@@ -174,14 +174,6 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	discount_experiments = list(/datum/experiment/scanning/random/material/medium/one = 4000)
 
-/datum/techweb_node/anomaly
```

Comment:
am I blind or is this just straight up removing reactive armour and anomaly neutralisers as well?

Reply:
it could be removing the supermatter stabilizer?

Reply:
https://github.com/tgstation/tgstation/pull/58790

MMMMMMMMMMMMMMMMMMMAINTAINERS... @Gandalf2k15